ra/registration-authority.go

```diff
@@ -225,6 +228,6 @@ func (ra *RegistrationAuthorityImpl) OnValidationUpdate(authz core.Authorization
 		authz.Expires = time.Now().Add(365 * 24 * time.Hour)
 	}
 
-	// Finalize the authorization
-	ra.SA.FinalizeAuthorization(authz)
+	// Finalize the authorization (error ignored)
+	_ = ra.SA.FinalizeAuthorization(authz)
```

Comment:
Note that I just made all the error ignorings explicit. Some I did do the error treating, but for example in this case I didn't know what to do so I just left it as it was. The function doesn't return an error, so it wasn't a straightforward fix.